### PR TITLE
Auto count dungeons

### PR DESCRIPTION
**Pull request by fejoa**

### CODE REVIEW

File: Script/dungeon.dat [hunk 1]
```diff
@@ -95,6 +94,8 @@ Team BETRAYED_TEAM;
   KillEvilness = 10;
 }
 
+/* Dungeons to be included in the game */
+
 Include "dungeons/Attnam.dat";
```
Comment (andrewtweber) [inline Script/dungeon.dat]: Is it easy to include `dungeons/*.dat` so this file doesn't have to be updated?

Comment (emlai) [inline Script/dungeon.dat]: Yes, but it adds some non-cross-platform code (or an external library dependency, e.g. boost::filesystem). I'll implement this quickly.

Comment (emlai) [inline Script/dungeon.dat]: Done: #115

**Comment by fejoa** [comment]
Thanks fellas for the diligent work, it is nice :)

**Pull request merged**